packages/hint-create-element-svg/README.md

```diff
@@ -0,0 +1,80 @@
+# No `createElement` with SVG (`create-element-svg`)
+
+This hint informs users that they need to use [`createElementNS`][createElementNS] to create SVG elements instead of [`createElement`][createElement]
+
+## Why is this important?
+
+XML documents may contain elements and attributes that are defined for and used by multiple softwares.
```

Comment:
Discussion of `XML` may be confusing here as most developers will get this warning when building `HTML` documents. Here's some proposed wording that focuses more on `HTML` instead.
```md
SVG-in-HTML is a fantastic addition to the web platform, but since SVG
is an XML-based language there is some nuance to how it can be used.
When parsing HTML, SVG elements are automatically created correctly so
long as they are inside an `<svg>...</svg>` block. However, SVG elements
cannot be *dynamically* created using `createElement` in the same way
as HTML elements.

For example, calling `document.createElement('circle')` actually returns
an *HTML* element named `circle` instead of an SVG element. This is for
compatibility with existing web content which may have created custom
(although invalid) HTML elements using these same names.

In order to dynamically create SVG elements, you must explicitly tell the
browser you want SVG by using the [SVG namespace][svg namespace] with
`createElementNS`. For example, to create an SVG `circle` call
`document.createElementNS('http://www.w3.org/2000/svg', 'circle')`.
```